Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1654,8 +1654,7 @@ protected static function initRefinery()
 	{
 		$container['refinery'] = function ($container) {
 			$dataFactory = new \ILIAS\Data\Factory();
-			$validationFactory = new \ILIAS\Refinery\Validation\Factory($dataFactory, $container->language());
-			return new \ILIAS\Refinery\Factory($validationFactory);
+			return new \ILIAS\Refinery\Factory($dataFactory);
```

Comment:
@legionth / @klees  : Should we put the `\ILIAS\Data\Factory` into the `DIC` as well? Or do we want to hide this from public and only provide implicit access/usage via the `Refinery`. Any opinions?

Reply:
Isn't it in the `DIC` already? The UI-Framework uses it as well. If not, I would put it there. If people want to depend on it (like Refinery and UI-Framework) that's fine for me.

Reply:
No, it's not, yet.